### PR TITLE
[15.0] [MIG] product_supplierinfo_archive

### PR DESCRIPTION
**Pull request by JasminSForgeFlow**
Standard Migration

@ForgeFlow

### CODE REVIEW

File: product_supplierinfo_archive/i18n/product_supplierinfo_archive.pot [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* product_supplierinfo_archive
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 13.0\n"
```
Comment (MiquelRForgeFlow) [inline product_supplierinfo_archive/i18n/product_supplierinfo_archive.pot]:
```suggestion
"Project-Id-Version: Odoo Server 15.0\n"
```

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by AaronHForgeFlow — APPROVED** [review]
Functional review LG!

**Pull request merged**